### ISSUE TITLE
Change Twitter log to current logo

### ISSUE THREAD

**Issue by shindodkar** [wagtail/wagtail.org#398]
-change to twitter logo to new logo(X)
 
![Screenshot (19)](https://github.com/wagtail/wagtail/assets/101176974/6215a7f5-3ee7-4dfe-b4c7-c3b57dca03ff)

**Comment by zerolab** [comment]
I don't know what the engagement stats are, but to be honest, my preference would be to either drop it entirely, or link to Mastodon.

**Comment by oly-baby** [comment]
@thibaudcolas pls can i work on this

**Comment by zerolab** [comment]
There's #423. Would be worth if you could test it.

**Comment by thibaudcolas** [comment]
This is our most used social media channel by far so let’s not drop it just because it’s annoying to have to change the logo.

**Comment by Scotchester** [comment]
> This is our most popular social media channel by far so let’s not drop it just because it’s annoying to have to change the logo.

I don't _think_ that's why Dan was interested in dropping it. If he's like me, it's due to the toxic leadership of the platform at this time. It is definitely not a simple choice for an entity like Wagtail to drop it, though, if it's where we get the most engagement.

**Comment by thibaudcolas** [comment]
@Scotchester fair enough :) Dan was commenting on the engagement stats on an issue about replacing the logo so that’s why I replied along those lines. I think a discussion about the toxic leadership at Twitter/X definitely has its place, but not on a ticket about keeping the logo up to date.

As far as engagement, for what it’s worth I don’t think we’re committed to any one platform necessarily. If you want to discuss this further I’d suggest #developer-relations on Slack, as the team there has spent lots of time discussing a lot of the options out there.
